includes/Admin.php

```diff
@@ -1355,20 +1364,25 @@ public function add_product_variation_edit_fields( $index, $variation_data, $pos
 			)
 		);
 
-		woocommerce_wp_textarea_input(
+		echo '<div class="wp-editor-wrap">';
+		echo '<label for="' . esc_attr(\WC_Facebookcommerce_Integration::FB_PRODUCT_DESCRIPTION) . '">' . 
+			 esc_html__( 'Facebook Description', 'facebook-for-woocommerce' ) . 
+			 '</label>';
+		wp_editor(
+			$rich_text_description,
```

Comment:
this is the major reason for preserving rich text description as on the ui, we need to render the text in the box with styling added